src-ui/lib/components/BooksTable.svelte

```diff
@@ -71,30 +81,34 @@
 {/await}
 
 <svelte:document use:listenForBarcodes on:scan={handleScan} />
-<table class="table is-fullwidth">
-  <thead>
-    <tr>
-      <th>ISBN</th>
-      <th>Title</th>
-      <th>Author</th>
-      <th>Tags</th>
-      <th>Read?</th>
-    </tr>
-  </thead>
-  <tbody>
-    {#each books.value as book}
-      <BooksTableRow {book} {handleEdit} />
-    {:else}
+{#await booksStorePromise}
+  ...initial loading of books...
+{:then booksStore}
```

Comment:
Since the DB needs to be waited on, we are very briefly rendering a placeholder.